src/plone/restapi/services/workflow/info.py

```diff
@@ -68,8 +68,20 @@ def __call__(self, expand=False):
                 title = title.decode("utf8")
             history[item]["title"] = self.context.translate(title)
 
+        current_state = wftool.getInfoFor(self.context, "review_state")
+
         result["workflow"].update(
-            {"history": json_compatible(history), "transitions": transitions}
+            {
+                "history": json_compatible(history),
+                "transitions": transitions,
+                "state": {
+                    "id": current_state,
+                    "title": wftool.getTitleForStateOnType(
+                        current_state,
+                        self.context.portal_type,
+                    ),
```

Comment:
I think that `title` should go through `context.translate` like in line 69 to return its translated title.